packages/camera/camera/example/pubspec.yaml

```diff
@@ -9,23 +9,24 @@ dependencies:
     # The example app is bundled with the plugin so we use a path dependency on
     # the parent directory to use the current plugin's version.
     path: ../
-  path_provider: ^0.5.0
+  path_provider: ^2.0.0
   flutter:
     sdk: flutter
-  video_player: ^0.10.0
-  integration_test:
-    path: ../../../integration_test
+  video_player: ^2.0.0
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
   flutter_driver:
     sdk: flutter
-  pedantic: ^1.8.0
+  integration_test:
+    path: ../../../integration_test
+  test: any
```

Comment:
Was this to resolve a conflict? We don't usually use `any` unless we have to; I would expect `1.16.0` (or 1.16.3 if you need `Fake`).

Reply:
This was actually a mistake on my end. I added this dependency because I was receiving a CI failure with the dependencies related to the `integration_test` in combination with running in NNBD. I started comparing with the `video_player` plugin and noticed that this dependency was included this way and decided to give that a try. Reading back in the Discord channel I realised that we shouldn't run integration_tests in NNBD mode (and add the `// @dart = 2.9` instruction to the top of the file). Unfortunately I forgot to remove this dependency which I did this time ;).